src/MPI.jl

```diff
@@ -32,32 +35,22 @@ function __init__()
         end
     end
 
-    global const mpitype_dict = Dict{DataType, Cint}(
-        # Older versions of OpenMPI (such as those used by default in
```

Comment:
okay, pushed a commit restoring the comment (though for some reason it is taking a while to show up on github)